ddtrace/tracer.py

```diff
@@ -480,6 +477,18 @@ def _check_new_process(self):
 
         self._pid = pid
 
+        ctx = self.get_call_context()
+        # The spans remaining in the context can not and will not be finished
+        # in this new process. So we need to copy out the trace metadata needed
+        # to continue the trace.
+        # Also, note that because we're in a forked process, the lock that the
+        # context has might be permanently locked so we can't use ctx.clone().
+        self.context_provider.activate(Context(
```

Comment:
```suggestion
        self.context_provider.activate(ctx.clone())
```

Should do the same thing.

Reply:
Please see the comment on line 484.

Reply:
I can't read I guess :)